The package builds successfully with `nix-build -E 'with import <nixpkgs> {}; callPackage ./default.nix {}'`, but i got this error while running the package

```python
Nerd Fonts Patcher v3.0.0 (4.1.1) (ff 20230101)
Traceback (most recent call last):
  File "/nix/store/8zxmwbjkbb4s5f217ll4lkql0pw7zn5z-nerd-font-patcher-3.0.0/bin/.nerd-font-patcher-wrapped", line 1996, in <module>
    main()
  File "/nix/store/8zxmwbjkbb4s5f217ll4lkql0pw7zn5z-nerd-font-patcher-3.0.0/bin/.nerd-font-patcher-wrapped", line 1951, in main
    args = setup_arguments()
  File "/nix/store/8zxmwbjkbb4s5f217ll4lkql0pw7zn5z-nerd-font-patcher-3.0.0/bin/.nerd-font-patcher-wrapped", line 1859, in setup_arguments
    logger.critical("FontnameParser module missing (bin/scripts/name_parser/Fontname*), specify --makegroups 0")
NameError: name 'logger' is not defined
```

Which i assume is wrong, as the python file does have `import logging` line.

I assume that this happens because I only ran `nix-build`, but if I git clone and run the script manually it does not give this error.

I suspect this wont happen if i install it in shell or on system.

Please do check and suggest me if there is something wrong, or for changes to be made!

--- a/pkgs/applications/misc/nerd-font-patcher/default.nix
+++ b/pkgs/applications/misc/nerd-font-patcher/default.nix
@@ -1,19 +1,13 @@
-{ python3Packages, lib, fetchFromGitHub }:
+{ python3Packages, lib, fetchzip }:
 
 python3Packages.buildPythonApplication rec {
   pname = "nerd-font-patcher";
-  version = "2.2.2";
+  version = "3.0.1";
 
-  # This uses a sparse checkout because the repo is >2GB without it
-  src = fetchFromGitHub {
-    owner = "ryanoasis";
-    repo = "nerd-fonts";
-    rev = "v${version}";
-    sparseCheckout = [
-      "font-patcher"
-      "/src/glyphs"
-    ];
-    sha256 = "sha256-boZUd1PM8puc9BTgOwCJpkfk6VMdXLsIyp+fQmW/ZqI=";
+  src = fetchzip {
+    url = "https://github.com/ryanoasis/nerd-fonts/releases/download/v${version}/FontPatcher.zip";
+    sha256 = "sha256-kh3zQ0lXCmiO72ZXwvLm7LGUZu0hg8G8TG+pknkK1yo=";
+    stripRoot = false;
   };
 
   propagatedBuildInputs = with python3Packages; [ fontforge ];
@@ -22,15 +16,19 @@ python3Packages.buildPythonApplication rec {
 
   postPatch = ''
     sed -i font-patcher \
-      -e 's,__dir__ + "/src,"'$out'/share/nerd-font-patcher,'
+      -e 's,__dir__ + "/src,"'$out'/share/,'
+    sed -i font-patcher \
+      -e  's,/bin/scripts/name_parser,/../lib/name_parser,'
   '';
+  # Note: we cannot use $out for second substitution
 
   dontBuild = true;
 
   installPhase = ''
-    mkdir -p $out/bin $out/share/nerd-font-patcher
+    mkdir -p $out/bin $out/share $out/lib
     install -Dm755 font-patcher $out/bin/nerd-font-patcher
-    cp -ra src/glyphs $out/share/nerd-font-patcher
+    cp -ra src/glyphs $out/share/
+    cp -ra bin/scripts/name_parser $out/lib/
   '';
 
   meta = with lib; {

